server/v2/flags.go

```diff
@@ -10,5 +10,9 @@ const (
 	FlagLogNoColor = "log_no_color"
 	FlagTrace      = "trace"
 
-	OutputFormatJSON = "json"
+	OutputFormatJSON      = "json"
+	FlagPruning           = "pruning"
```

Comment:
can we move those flags closer to where they are used?

Reply:
Yes forgot about it